### PR TITLE
Feature/50 add validations for maven release

### LINKED ISSUE #50: Add a pom file validation for Maven Release

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #50

### CODE REVIEW

File: src/test/java/com/exasol/releaserobot/maven/MavenPlatformValidatorTest.java [hunk 1]
```diff
@@ -7,34 +7,96 @@
...
+    void testValidatePGpgPluginMissingExecutions() {
+        when(this.mavenPomMock.getPlugins())
+                .thenReturn(List.of(MavenPlugin.builder().artifactId("maven-gpg-plugin").build()));
+        final Report report = this.platformValidator.validate(this.repositoryMock);
+        assertAll(() -> assertThat(report.hasFailures(), equalTo(true)),
+                () -> assertThat(report.getFailuresReport(), containsString("E-RR-VAL-14")));
+    }
+
+    @Test
+    void testValidatePGpgPluginMissingRequiredExecution() {
```
Comment (chiaradiamarcelo) [inline src/test/java/com/exasol/releaserobot/maven/MavenPlatformValidatorTest.java:82]:
```suggestion
    void testValidateGpgPluginMissingRequiredExecution() {
```

File: doc/user_guide/maven_central_release_template.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Template of `maven_central_release.yml`
+
+Copy the content into a file `/.github/workflows/maven_central_release.yml` in your project.
```
Comment (chiaradiamarcelo) [inline doc/user_guide/maven_central_release_template.md:3]:
```suggestion
Create a `/.github/workflows/maven_central_release.yml` file in your project with the following content:
```

**Review by chiaradiamarcelo — CHANGES_REQUESTED** [review]

File: doc/user_guide/user_guide.md [hunk 3]
```diff
@@ -70,13 +71,27 @@ Rules:
...
- This line should appear between the file's header and the first section describing the changes.
+  This line should appear between the file's header and the first section describing the changes.
 
 * A `changes_<version>.md` file must contain one or more GitHub ticket numbers in the following format: '#1:<comment>'.
   All the mentioned on the file tickets must have a closed status on the GitHub.
+ 
+### Rules for Release on Maven Central
+
+Prerequisites:
+
```
[the remark this answers is not in the document]
Comment (AnastasiiaSergienko) [reply doc/user_guide/user_guide.md:85]: Moved those rules to ### Common Rules for All Repositories

File: src/main/java/com/exasol/releaserobot/maven/MavenPlatformValidator.java [hunk 4]
```diff
@@ -19,9 +24,74 @@ public Report validate(final Repository repository) {
         return report;
```
[the remark this answers is not in the document]
Comment (chiaradiamarcelo) [reply src/main/java/com/exasol/releaserobot/maven/MavenPlatformValidator.java:24]: Agreed to deal with in another PR

**Review by chiaradiamarcelo — APPROVED** [review]

**Pull request merged**